float32 wheel1_velocity
float32 wheel1_setpoint
float32 wheel1_output
float32 wheel1_position
float32 wheel1_i_accum

float32 wheel2_velocity
float32 wheel2_setpoint
float32 wheel2_output
float32 wheel2_position
float32 wheel2_i_accum

float32 wheel3_velocity
float32 wheel3_setpoint
float32 wheel3_output
float32 wheel3_position
float32 wheel3_i_accum

float32 wheel4_velocity
float32 wheel4_setpoint
float32 wheel4_output
float32 wheel4_position
float32 wheel4_i_accum